images/wordpress/Dockerfile

```diff
@@ -1,25 +1,24 @@
 FROM php:apache
 
-# APT
 RUN apt-get update && apt-get install -y --no-install-recommends libpng12-dev vim-tiny less mysql-client && rm -r /var/lib/apt/lists/*
 
-# PHP/Apache setup
 RUN docker-php-ext-install mysqli && \
     docker-php-ext-install gd && \
     a2enmod rewrite
 
-# wp-cli
-RUN curl --silent https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar > /usr/local/bin/wp-cli.phar &&\
-    /bin/echo -e '#!/bin/sh\nexec php /usr/local/bin/wp-cli.phar --allow-root "${@}"' > /usr/local/bin/wp &&\
-    chmod 755 /usr/local/bin/wp
+RUN curl --silent https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar > /usr/local/bin/wp-cli.phar
 
-# WordPress
-RUN curl --silent https://wordpress.org/latest.tar.gz > /usr/src/latest.tar.gz && \
-    tar -C /usr/src -xzf /usr/src/latest.tar.gz && \
-    rm -rf /var/www/html /usr/src/wordpress/wp-content && \
-    mv /usr/src/wordpress /var/www/html
+ENV WORDPRESS_VERSION=latest
+
+COPY wp /usr/local/bin/
+COPY wp-start /usr/local/bin/
+
+RUN wp core download --path=/var/www/html
```

Comment:
`wp core download --path=/var/www/html` checks what the latest version is, and will only download a new version of WP if the current version is not the latest.

`wp core download --path=/var/www/html --version=${WORDPRESS_VERSION} --force` (note the `--force` option) always downloads a copy of WordPress, even if it's the same version as what's already installed.

I should add a brief comment explaining the behaviour.